connector/src/main/resources/languages/en_us.properties

```diff
@@ -0,0 +1,117 @@
+#
+# Copyright (c) 2019-2020 GeyserMC. http://geysermc.org
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to deal
+#  in the Software without restriction, including without limitation the rights
+#  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#  copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+#
+#  @author GeyserMC
+#  @link https://github.com/GeyserMC/Geyser
+#
+#
+
+geyser.config.failed=Failed to read/create config.yml! Make sure it's up to date and/or readable+writable!
+
+geyser.core.load=Loading {0} version {1}
+geyser.core.start=Started Geyser on {0}:{1}
+geyser.core.fail=Failed to start Geyser on {0}:{1}
+geyser.core.done=Done ({0})! Run /geyser help for help!
+geyser.core.shutdown=Shutting down Geyser.
+geyser.core.shutdown.kick.log=Kicking {0} player(s)
+geyser.core.shutdown.kick.message=Geyser Proxy shutting down.
+geyser.core.shutdown.kick.done=Kicked all players!
+geyser.core.shutdown.done=Geyser shutdown successfully.
+
+geyser.entity.type_null=Entity type {0} was null.
+geyser.entity.player.spawn=Spawned player at {0} {1} {2}
+geyser.entity.player.failed_list=Haven't received PlayerListEntry packet before spawning player! We ignore the player {0}
+
+geyser.particle.failed_map=Fail to map particle {0}=>{1}
+
+geyser.language.missing_file=Missing Geyser language file: {0}
+geyser.language.load_failed=Unable to load Geyser locale map for {0}
+
+geyser.locale.fail.latest_version=Unable to get latest Minecraft version
+geyser.locale.fail.asset_cache=Failed to load locale asset cache: {0}
+geyser.locale.fail.invalid=Invalid locale requested to download and load: {0}
+geyser.locale.fail.file=Unable to load locale: {0} ({1})
+geyser.locale.fail.json=Unable to load Java edition locale map for {0}
+geyser.locale.fail.missing=Missing MC locale file: {0}
+geyser.locale.fail.en_us=Unable to download and extract en_us locale!
```

Comment:
Probably makes more sense for this to be `geyser.locale.en_us.download`.

Reply:
oops just seen this, is it worth making the change now?